crates/bevy_gizmos/src/animated/mod.rs

```diff
@@ -0,0 +1,204 @@
+//! A module for the [`AnimatedGizmos`] [`SystemParam`].
+
+use std::ops::{Deref, DerefMut};
+
+use bevy_color::Color;
+use bevy_ecs::system::{Res, SystemParam};
+use bevy_math::Vec3;
+use bevy_time::Time;
+
+use crate::prelude::{DefaultGizmoConfigGroup, GizmoConfigGroup, Gizmos};
+
+/// A [`SystemParam`] for drawing animated gizmos.
```

Comment:
```suggestion
/// A [`SystemParam`] for drawing animated gizmos, whose segments cycle along the line.
```

This is one very specific type of animation: it's good to explain up front what it does.